src/erasure-code/shec/ErasureCodeShec.cc

```diff
@@ -414,7 +414,7 @@ void ErasureCodeShecReedSolomonVandermonde::prepare()
   dout(10) << " [ technique ] = " <<
     ((technique == MULTIPLE) ? "multiple" : "single") << dendl;
 
-  ceph_assert((technique == SINGLE) || (technique == MULTIPLE));
+  ceph_assert((technique == SINGLE)) || ceph_assert((technique == MULTIPLE));
```

Comment:
@bridg8et changes looks good overall, just a small change: many places we have not needed double quotes, can you work on those instances eg: ceph_assert((technique == SINGLE))  -> ceph_assert(technique == SINGLE)